### PR TITLE
Updated authors doc

### PR DESCRIPTION
**Pull request by RichardLitt**
License: MIT
Signed-off-by: Richard Littauer <richard.littauer@gmail.com>

### CODE REVIEW

**Comment by Kubuxu** [comment]
You have Jeremy three times on this list.

Also could you use my other email address: kubuxu@protonmail.ch

**Comment by RichardLitt** [comment]
The list is automatically generated. Should we remove duplicates? Should we add multiple emails more clearly?

**Comment by dignifiedquire** [comment]
If the tool to generate these uses `git` under the hood you can use a `.mailmap` to alias multiple emails, see https://github.com/karma-runner/karma/blob/master/.mailmap for an example

**Comment by Kubuxu** [comment]
I think we already had tool that handled it for generating weekly reports.
It was working great.

**Comment by RichardLitt** [comment]
@Kubuxu Which tool? You mean https://github.com/RichardLitt/name-your-contributors?

**Comment by Kubuxu** [comment]
Hmm, I thought about this one: https://github.com/ipfs/newsletter/blob/master/tools/get_commits.py

But I think yours is superior.

Simple mapping of email to alternative email would be enough to fix most cases.
(I just noticed that I am on this list 3 times).

**Comment by RichardLitt** [comment]
Ah! Well, @eminence's tool `get_commits.py` is awesome, and part of mine. I need to make the javascript version of that. 

I'll try and prioritize making a js commit version (been overloaded recently) to make this better.

**Comment by eminence** [comment]
Having two tools that do almost the same thing is kinda unfortunate, so moving the functionality of get_commits.py into name-your-contributors is a good goal

**Comment by RichardLitt** [comment]
@eminence I'm on it, just hasn't been as prioritized as it should be as yours works, right now, and I've got other stuff. Will hopefully move it up!

**Comment by RichardLitt** [comment]
For now, this is probably good to go?

**Comment by Kubuxu** [comment]
Let me remove two of myself and one Juan.

**Comment by eminence** [comment]
Alas, we have yet to figure out how to clone @Kubuxu and the rest of the team.  If only we could, it would  be a huge boon to productivity!

**Comment by Kubuxu** [comment]
LGTM

**Comment by Kubuxu** [comment]
LGTM

File: docs/AUTHORS [hunk 1]
```diff
@@ -2,70 +2,111 @@
...
 Simon Kirkby <tigger@interthingy.com>
 Siraj Ravel <jason.ravel@cbsinteractive.com>
 Siraj Ravel <llSourcell@gmail.com>
+Siva Chandran <siva.chandran@realimage.com>
+slothbag <slothbag>
+sroerick <sroerick@gmail.com>
 Stephan Seidt <evilhackerdude@gmail.com>
 Stephan Seidt <stephan.seidt@gmail.com>
 Stephen Sugden <me@stephensugden.com>
+Stephen Whitmore <noffle@users.noreply.github.com>
```
Comment (whyrusleeping) [inline docs/AUTHORS]: still a duplicate here

File: docs/AUTHORS [hunk 2]
```diff
@@ -2,70 +2,111 @@
...
 Lars Gierth <larsg@systemli.org>
 llSourcell <sirajravel@gmail.com>
+Marcin Janczyk <marcinjanczyk@gmail.com>
 Marcin Rataj <lidel@lidel.org>
 Markus Amalthea Magnuson <markus.magnuson@gmail.com>
 Matt Bell <mappum@gmail.com>
+michael <pfista@gmail.com>
 Michael Lovci <michaeltlovci@gmail.com>
 Michael Muré <batolettre@gmail.com>
+Michael Muré <mure.michael@gmail.com>
```
Comment (whyrusleeping) [inline docs/AUTHORS]: this ones a dup

File: docs/AUTHORS [hunk 3]
```diff
@@ -2,70 +2,111 @@
...
+Michael Pfister <pfista@gmail.com>
 Mildred Ki'Lya <mildred-pub.git@mildred.fr>
 Muneeb Ali <muneeb@ali.vc>
+Nick Hamann <nick@wabbo.org>
+palkeo <contact@palkeo.com>
 Patrick Connolly <patrick.c.connolly@gmail.com>
 Pavol Rusnak <stick@gk2.sk>
 Peter Borzov <peter@sowingo.com>
 Peter Borzov <tihoutrom@gmail.com>
```
Comment (whyrusleeping) [inline docs/AUTHORS]: dup

File: docs/AUTHORS [hunk 4]
```diff
@@ -2,70 +2,111 @@
...
+Dominic Della Valle <DDVpublic@Gmail.com>
 Dominic Tarr <dominic.tarr@gmail.com>
 drathir <drathir87@gmail.com>
 Dylan Powers <dylan.kyle.powers@gmail.com>
 Emery Hemingway <emery@vfemail.net>
 epitron <chris@ill-logic.com>
 Ethan Buchman <ethan@coinculture.info>
 Etienne Laurin <etienne@atnnn.com>
+Forrest Weston <fweston@eecs.wsu.edu>
+ForrestWeston <Forrest.Weston@gmail.com>
```
Comment (whyrusleeping) [inline docs/AUTHORS]: dup

File: docs/AUTHORS [hunk 5]
```diff
@@ -2,70 +2,111 @@
...
 Chas Leichner <chas@chas.io>
 Chris Grimmett <xtoast@gmail.com>
+Chris P <sahib@online.de>
+Chris Sasarak <chris.sasarak@gmail.com>
 Christian Couder <chriscool@tuxfamily.org>
+Christian Kniep <christian@qnib.org>
+Christopher Sasarak <chris.sasarak@gmail.com>
 David <github@kattfest.se>
 David Braun <David.Braun@Toptal.com>
+David Dias <daviddias.p@gmail.com>
```
Comment (whyrusleeping) [inline docs/AUTHORS]: dup

**Comment by RichardLitt** [comment]
Will make a deduplicator.

File: docs/AUTHORS (hunk 3 above)
Comment (RichardLitt) [inline docs/AUTHORS]: How am I supposed to know that's a dup programmatically?

**Comment by RichardLitt** [comment]
Just did it manually. Should be good to go.

**Pull request merged**